src/PostProcess/RenderPipeline/Pipelines/babylon.standardRenderingPipeline.ts

```diff
@@ -282,6 +327,13 @@
             this.textureAdderFinalPostProcess = new PostProcess("HDRDepthOfFieldSource", "standard", [], [], ratio, null, Texture.BILINEAR_SAMPLINGMODE, scene.getEngine(), false, "#define PASS_POST_PROCESS", Engine.TEXTURETYPE_UNSIGNED_INT);
             this.addEffect(new PostProcessRenderEffect(scene.getEngine(), "HDRBaseDepthOfFieldSource", () => { return this.textureAdderFinalPostProcess; }, true));
 
+            // Create volumetric light

```

Comment:
I'm wondering if postprocesses should be created ONLY if required (on first call?)